<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="inputWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>416</width>
          <height>360</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>416</width>
          <height>360</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <property name="leftMargin">
          <number>72</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>72</number>
         </property>
         <property name="bottomMargin">
          <number>44</number>
         </property>
         <item>
          <widget class="QWidget" name="title" native="true">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>40</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="minimumSize">
               <size>
                <width>196</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>196</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>InteractiveClass.Title</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="inputArea">
           <property name="minimumSize">
            <size>
             <width>271</width>
             <height>280</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>271</width>
             <height>280</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_1">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>14</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>InteractiveClass.InputClassNo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="classNumWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>271</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>271</width>
                <height>40</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="classRoomIdEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="contextMenuPolicy">
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="cursorPosition">
                  <number>0</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>14</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>InteractiveClass.InputUserNick</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="classUserNickWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>271</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>271</width>
                <height>40</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="classUserNickEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="contextMenuPolicy">
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="cursorPosition">
                  <number>0</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>14</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>14</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>InteractiveClass.InputUserNo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="userWidget" native="true">
              <property name="minimumSize">
               <size>
                <width>271</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>271</width>
                <height>40</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="userIdEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="contextMenuPolicy">
                  <enum>Qt::NoContextMenu</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="cursorPosition">
                  <number>0</number>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget" native="true">
              <property name="minimumSize">
               <size>
                <width>271</width>
                <height>39</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>271</width>
                <height>39</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="loginBtn">
                 <property name="minimumSize">
                  <size>
                   <width>271</width>
                   <height>39</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>271</width>
                   <height>39</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>InteractiveClass.DoLogin</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="demo.qrc"/>
 </resources>
 <connections/>
</ui>
